assets/src/design-system/components/slider/index.js

```diff
@@ -284,12 +284,16 @@ function Slider({
 Slider.propTypes = {
   minorStep: PropTypes.number.isRequired,
   majorStep: PropTypes.number.isRequired,
-  handleChange: PropTypes.func.isRequired,
+  handleChange: PropTypes.func,
   value: PropTypes.number.isRequired,
   min: PropTypes.number,
   max: PropTypes.number,
   thumbSize: PropTypes.number,
   suffix: PropTypes.string,
 };
 
+Slider.defaultProps = {
+  handleChange: () => {},
```

Comment:
Instead of using `.defaultProps`, should just set the default value in the function signature (like for all the other props there).

Also, should use thee `noop` util there. So:

```js
function Slider({
  minorStep,
  majorStep,
  handleChange = noop,
  value,
  min = 0,
  max = 500,
  thumbSize = DEFAULT_SIZE,
  suffix = '',
  ...rest
}) {
// ...
```